lib/utils/remote-build.ts

```diff
@@ -98,7 +98,7 @@ export async function startRemoteBuild(build: RemoteBuild): Promise<void> {
 			output: process.stdout,
 		});
 
-		rl.on('SIGINT', () => process.emit('SIGINT'));
+		rl.on('SIGINT', () => process.emit('SIGINT' as any));
```

Comment:
At least when `"@types/node"` is `"10.14.5"`, I'd get:
```
lib/utils/remote-build.ts:101:38 - error TS2345: Argument of type '"SIGINT"' is not assignable to parameter of type '"disconnect"'.
101  	rl.on('SIGINT', () => process.emit('SIGINT'));
```
The emitter is typed:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/df175aa50bc88447a9e1fb15d9a6b9c89c50246b/types/node/globals.d.ts#L937-L948

If could add an argument to get type checking to pass, something like:
```
rl.on('SIGINT', () => process.emit('SIGINT', 'SIGINT'));
```
But I wasn't sure of the runtime implications of that change -- I would have to test and it isn't easy, and it's supposed to be working as it is, so I thought it would be safer to skip type checking with `as any`.